pkg/crypto/tinkcrypto/cl_crypto.go

```diff
@@ -0,0 +1,316 @@
+//go:build ursa
+// +build ursa
```

Comment:
it looks like ursa tags are not included in our linter

Reply:
seems it'll take `ursa` and `cgo` to be provided in `golangci-lint` Docker container. 
should we create a new image for lint with these dependencies inside and use it in `check_lint.sh`?

Reply:
Yeah, it seems the default golangci-lint won't have these tools, might as well create a new dedicated image. Thanks.

Reply:
@Baha-sk Where should I keep related `Dockerfile` and push the corresponding image?
Also, we could try to make a separate github job as Alex did for ursa tests - it'll use docker with pre-installed `ursa`

Reply:
you can create a `ursa` subfolder under `images` and have a new dockerfile there.

as for the job, i would recommend simply updating `scripts/check_lint.sh` with new entries for `ursa` linting using your new image.

Reply:
@Baha-sk can we postpone this chage to the next PR?
Currently, I'm not a contributor to af-go, since I haven't made any commit to the repo. Therefore, pipelines would be not started automatically. And it'll be painful to ask you to start pipeline on each of me changes during debug.
Although, I ran `golangci-lint` locally with `ursa` tag and made the fixes according to all lint warnings.

Reply:
no issues @kgoncharov, this is not mandatory, but a suggestion if you need to have a separate dockerfile for ursa builds.